plugins/in_windows_exporter_metrics/we_util.c

```diff
@@ -54,10 +53,12 @@ double we_get_windows_version()
 
     if (result != ERROR_SUCCESS)
     {
-        return 0;
+        return -1;
     }
 
-    return strtod(version_text, NULL);
+    *version_number = strtod(version_text, NULL);
+
+    return 0;
```

Comment:
We were using 0 as an error return before but now a success return so want to check this is correct? Should we have enums for this?

Reply:
Good point. Using `FLB_TRUE` or `FLB_FALSE` would be better.

Reply:
I use `FLB_TRUE` and `FLB_FALSE` for returning values here. This should be cleaner whether error or not.
Fixed in https://github.com/fluent/fluent-bit/pull/6621/commits/5c5e5f16d976845d883ab382020c936de30dcf83.